arweave/caip122.md

```diff
@@ -0,0 +1,71 @@
+---
+namespace-identifier: arweave-caip122
+title: Arweave Namespace - SIWx
+author: Rohit Pathare (@ropats16), Phil Billingsby (@pbillingsby), Dan MacDonald (@DanMacDonald)
+discussions-to: TBA
+status: Draft
+type: Standard
+created: 2022-09-01
+requires: ["CAIP-122", "CAIP-2", "CAIP-10"]
+---
+
+## CAIP-122
+
+For context, see the [CAIP-122](CAIP-122) specification.
+
+## Rationale
+
+On Arweave, [EIP-4361](EIP-4361) Sign-in with Arweave already exists, which was the inspiration for CAIP-122. This specification highlights how EIP-4361 conforms with CAIP-122.
```

Comment:
does a sign in with arweave standard already exist? this references eth eip

Reply:
Arweave has not needed a sign in standard in the past because of the way the ecosystem works. However, with this integration, there is a use for it. So this CAIP is the first version of the standard. Would love some insights on how we can rephrase the rationale based on this information.

Reply:
yup makes sense, could usual similar wording to here - https://github.com/ChainAgnostic/namespaces/blob/main/solana/caip122.md and mention inspired by eip-4316, or reference